tests/p_abrt/0-install_abrt.sh

```diff
@@ -4,7 +4,7 @@
 
 if (t_GetPkgRel basesystem | grep -q el6)
 then
-    t_InstallPackage  abrt
+    t_InstallPackage  abbbbbbrt
```

Comment:
This is a failure introduced for testing purposes and won't be present non WIP pr